src/lib.rs

```diff
@@ -104,6 +104,8 @@ pub struct BatchLCProof<F: Field, PC: PolynomialCommitment<F>> {
 /// of evaluation for the corresponding commitments at a query set `Q`, while
 /// enforcing per-polynomial degree bounds.
 pub trait PolynomialCommitment<F: Field>: Sized {
+    /// Seed string used for initial setup
+    const PROTOCOL_NAME: &'static [u8];
```

Comment:
Mmm I would remove this from the trait and keep it in the implementers (like it was before)

Reply:
(It's used for parameters generation anyway, not used for Sponge)

Reply:
Ah ok I see you use them in the generic PolynomialCommitment tests. Forget it you can use whathever you want as initialization. Maybe would be useful (for this cases and for completeness) to introduce another constructor in the trait not taking any seed